### PR TITLE
Adds investment cost, lifetime, and e-waste of transportation sector

### LINKED ISSUE #120: Add investment cost to the transportation sector
This issue can be closed when the investment cost of EVs (Electric Vehicles) and FCEVs (Fuel Cell Electric Vehicles) is added to UIUC's latest model in Temoa.

### PR DESCRIPTION
**Pull request by robfairh**
Fixes #120, #124, and #125.

### CODE REVIEW

**Comment by robfairh** [comment]
#123 shows some of the outputs from the updated model.

File: temoa-uiuc/data_files/09_uiuc.sql [hunk 1]
```diff
@@ -594,6 +594,28 @@ INSERT INTO "EmissionActivity" VALUES ('uiuc', 'co2eq', 'E85','E85VCL', 2027,'UV
...
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2021,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2022,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2023,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2024,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2025,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2026,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2027,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2028,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2029,'UVCL',21.4,'kg/kGal','from waste calc');
+INSERT INTO "EmissionActivity" VALUES ('uiuc', 'ewaste', 'H2','H2VCL', 2030,'UVCL',21.4,'kg/kGal','from waste calc');
```
Comment (samgdotson) [inline temoa-uiuc/data_files/09_uiuc.sql:618]: What are you considering as ``EWASTE`` from hydrogen vehicles? Do they also use lithium ion batteries?

Comment (samgdotson) [reply temoa-uiuc/data_files/09_uiuc.sql:618]: I see, it's in the notebook

**Review by samgdotson — APPROVED** [review]
These changes look good! We can address the FixedCosts later

**Pull request merged**